src/util/cmdlineargs.cpp

```diff
@@ -152,6 +161,8 @@ void CmdlineArgs::printUsage() {
                         and spinning vinyl widgets. Try this option if\n\
                         Mixxx is crashing on startup.\n\
 \n\
+--noColors              Don't print colors on the console output.\n\
```

Comment:
Rather than a `--noColors` flag we should use `--color=never` or `always` or `auto` (`auto` checks if the process is attached to a TTY) like `ls`.

Reply:
good idea, done